src/app/components/SubjectHeading/SubjectHeadingShow.jsx

```diff
@@ -97,6 +97,7 @@ class SubjectHeadingShow extends React.Component {
         heading.children.some(child => this.removeChildrenOffMainPath(child, uuid))
       );
     if (!onMainPath) heading.children = null;
+    if (onMainPath) heading.onMainPath = true;
```

Comment:
Does this change the headings you can see by toggling, and if so, is that what we want?

Reply:
I don't quite understand the question. I found the to be the best way to insert a property to determine what are the appropriate headings to highlight.

Reply:
This is a new property added onto the subject headings from this component and it only effects styling.

Reply:
My mistake I thought line 99 was being removed, but it isn't.

Reply:
I thought that too for a second when responding to your question. So, I totally understand!